src/Classes/Executor.php

```diff
@@ -124,10 +126,28 @@ public function ping(string $url, array $headers = []): self
     /**
      * Handle the running of a console command.
      *
-     * @param  string  $commandToRun
+     * @param string $commandToRun
+     * @param bool $isInteractive
      */
-    private function runCommand(string $commandToRun): void
+    private function runCommand(string $commandToRun, bool $isInteractive = false): void
     {
+        if ($isInteractive) {
+            // This function is similar to exec(), However, it sends the raw output from the program to the output stream
+            // with which PHP is currently working (i.e. either HTTP in a web server scenario, or the shell in a command line version of PHP).
+            // for more info see: https://php.net/manual/en/function.passthru.php
+            // use escapeshellcmd() to return the command in a safe format that can be used.
+            passthru(escapeshellcmd($commandToRun), $status);
+            if ($status == 0) {
+                // success
+                $this->setOutput('Interactive command completed');
```

Comment:
This part seems to be overwriting the output that the Executor might already be holding. You'll likely have to update this line and line 145 to use something along the lines of:

```php
$this->setOutput($this->getOutput().'Interactive command completed');
```